motor.cpp

```diff
@@ -23,7 +23,22 @@ void Motor::stop(int motor, int motor2, int motor3, int motor4) {
     }
 }
 
+void Motor::go_all(
+    int motor1_speed, int motor2_speed,
```

Comment:
I think it would be better if this function took an array of speeds and another for directions (so that when we need to add/remove motors, we don't have to change it).